readthedocs/oauth/services/bitbucket.py

```diff
@@ -63,24 +67,26 @@ def sync_repositories(self):
                 "https://bitbucket.org/api/2.0/repositories/",
                 role="admin",
             )
-            admin_repo_relations = RemoteRepositoryRelation.objects.filter(
+            RemoteRepositoryRelation.objects.filter(
                 user=self.user,
                 account=self.account,
                 remote_repository__vcs_provider=self.vcs_provider_slug,
                 remote_repository__remote_id__in=[r["uuid"] for r in resp],
-            )
-            for remote_repository_relation in admin_repo_relations:
-                remote_repository_relation.admin = True
-                remote_repository_relation.save()
+            ).update(admin=True)
```

Comment:
Small optimization here, since we don't need to save each object individually.

Reply:
Also nice to avoid the `save()` logic if we don't need it.